.env.sample

```diff
@@ -1,7 +1,4 @@
-REACT_APP_CHAIN_ID="43114"
-REACT_APP_NETWORK_ID="1"
-REACT_APP_NETWORK_URL="https://api.avax.network/ext/bc/C/rpc"
-REACT_APP_MOONPAY_PK=""
-REACT_APP_COINBASE_PK=""
-REACT_APP_SUBGRAPH_BASE_URL="https://api.thegraph.com/subgraphs/name/pangolindex"
-REACT_APP_MIXPANEL=""
\ No newline at end of file
+VITE_MOONPAY_PK=""
+VITE_COINBASE_PK=""
+VITE_SUBGRAPH_BASE_URL="https://api.thegraph.com/subgraphs/name/pangolindex"
```

Comment:
Could we move these URL's to the SDK repository?

Reply:
yes @HariSeldon23